docs/sources/review/lint-prose/dictionary.md

```diff
@@ -0,0 +1,66 @@
+---
+description: Learn about the Grafana Labs dictionary template format.
+menuTitle: Dictionary
+review_date: 2024-06-04
+title: Grafana Labs' dictionary
+---
+
+# Grafana Labs' dictionary
+
+The Grafana Labs documentation squad maintains a dictionary used for spell checking.
+The same dictionary is also used to generate some Vale rules from the words metadata.
```

Comment:
Is this "word's" or "words" here?

Reply:
Maybe it's even "words'"?

Its the metadata belonging to each word for every word.

Reply:
Swap it around to the something like "the metadata in the word definition" or something like that?

Reply:
Yeah, much better, I'll incorporate that too when I check out this branch again